### ISSUE TITLE
wrong  Alos 16.1°

### ISSUE THREAD

**Issue by padfodnik** [KosherJava/zmanim#133]
In library and in site i get wrong Alos 16.1°

**Comment by KosherJava** [comment]
@padfodnik I would need a lot more information to help. When you say it is wrong, in reference to who/what? Please provide the lat/lng, the date, the algorithm used, and what you believe the correct time is, and the source for the time you believe to be correct.
Thanks

**Comment by padfodnik** [comment]
@KosherJava 
[zmanim_netanya.xlsx](https://github.com/KosherJava/zmanim/files/2624844/zmanim_netanya.xlsx)
`Location location1 = locationManager.getLastKnownLocation(provider);
        String locationName = "";
        double longitude = location1.getLongitude();
        double latitude = location1.getLatitude();
        Geocoder geocoder = new Geocoder(getApplicationContext(), Locale.getDefault());
        try {
            List<Address> listAddresses = geocoder.getFromLocation(latitude, longitude, 1);
            if(null!=listAddresses&&listAddresses.size()>0){
                locationName = listAddresses.get(0).getLocality()+", "+listAddresses.get(0).getCountryName();
            }
        } catch (IOException e) {
            e.printStackTrace();
        }

        double elevation = 0; //optional elevation
        TimeZone timeZone = TimeZone.getDefault();
        GeoLocation geoLocation = new GeoLocation(locationName, latitude, longitude, elevation, timeZone);`

zc.getAlosHashachar() and other methods returns wrong time

**Comment by KosherJava** [comment]
@padfodnik , I just tried my site and it works. Use " Asia/Jerusalem" as the timezone. Let me know if that works. Also see https://kosherjava.com/2011/11/08/outputting-zmanim-for-a-different-time-zone-with-the-the-zmanim-api/ . I did a quick test and the use of "Etc/GMT-2" and not "Etc/GMT+2" returns the correct output, but that is not the proper way to generate zmanim since at the minimum you will not have daylight savings time, and it also seems to be causing other issues. This is not a bug in the library. See 
https://www.reddit.com/r/java/comments/5i3zd1/timezoneid_etcgmt2_is_actually_gmt2/ that explains the issue you are facing.
